release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,79 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as more graceful upgrades, and improved worker labeling.
+
+### Important upgrade notes
+
+- Due to Azure limitations, upgraded clusters do not support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
+
+---
+
+### azure-operator v2.9.0
+
+#### Minor changes
+
+- Cluster scaling is less disruptive.
+
+#### Improved worker nodes' upgrade process
+
+With this release the worker nodes' upgrade process got some notable changes and it is now faster and more graceful. It works by mirroring the existing nodes with new ones and then draining the old nodes. Once all old nodes are drained, they are terminated. This way kubernetes controller manager and scheduler can move the running workload to new nodes while respecting configured scheduling directives for undisrupted service.
+
+In order to ensure continuous service during upgrade, the workload must be configured with multiple replicas and appropriate [pod disruption budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
+
+### Kubernetes v1.16.3
+
+- No change from the previous version
+
+### Calico v3.10.1
+
+- No change from the previous version
+
+### cert-exporter [v1.2.1](https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#121-2019-12-24)
+
+- No change from the previous version
+
+### chart-operator [v0.11.4](https://github.com/giantswarm/chart-operator/releases/tag/v0.11.4)
+
+- Now handling timeouts by pulling chart tarballs.
+
+### cluster-operator [v0.22.1](https://github.com/giantswarm/cluster-operator/releases/tag/v0.22.1)
+
+- Upgrade to Nginx Ingress Controller App 1.3.0
```

Comment:
```suggestion
- Upgrade to NGINX Ingress Controller App 1.3.0
```

just for uniformity, can we use one way of writing it? :) i saw NGINX write it all caps here https://github.com/kubernetes/ingress-nginx let me know if it's wrong